prompts/create-models.txt

```diff
@@ -359,4 +359,10 @@ export interface BookingModel {{
 
 ```yaml
 {api_definition}
-```
\ No newline at end of file
+```
+# Additional relevant context
+
+```additional context
+{additional_context}
+```
+
```

Comment:
Do you think additional context is needed for model generation?

Reply:
It is well known that LLMs outputs benefit from additional context, taking into consideration how these models can adjust different aspects of its responses based on instructions written in natural language, even accounting for subtlety.
Personally, I think that the more info you give to a model, the better.

You could, for example, add very specific modifications to a subset of services or models simply by specifying this in natural language.

A feature like this gives you the flexibility to adjust the testing framework to scenario-specific conditions or nuances.

I must admit that I do see the possible pitfalls of an implementation like this, giving the user direct access to a prompt can prove to be problematic.

I'm positive the current implementation can work as a starting point for new ideas and contributions.